src/test/christophermanahan/github/io/carnitas/SocketConnectionTest.java

```diff
@@ -0,0 +1,54 @@
+package christophermanahan.github.io.carnitas;
```

Comment:
Nice package name! FWIW, this is close to the convention, but not quite.

Following the convention would mean `io.github.christophermanahan.carnitas`